internal/run/consumer.go

```diff
@@ -116,6 +116,17 @@ func (c *Consumer) Consume(log *zap.Logger, repository *model.Repository) (err e
 
 	// parse and send to our API
 	for _, effxYAMLFile := range effxYAML {
+
+		// gets the dir where the effx file is at
+		// for example /src/stuff/effx.yaml -> /src/stuff/
+		effxDir := path.Join(effxYAMLFile, "./")
```

Comment:
We should use filepath.Base / path.Base so the CLI is platform independent. This forces linux...

Reply:
oh, good to know!